src/JsonHelper.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-nocheck
+import { Schema, SchemaHelper, ConfluentSubject, ConfluentSchema } from './@types'
+
+export default class JsonHelper implements SchemaHelper {
+  public validate(schema: Schema): void {
+    return
+  }
+
+  public getSubject(
+    confluentSchema: ConfluentSchema,
+    schema: Schema,
+    separator: string,
+  ): ConfluentSubject {
+    throw Error('not implemented yet')
```

Comment:
Did you have a chance to look into what the Java client does for this and Protobuf?

Reply:
If not, and we leave them unimplemented for now, let's at least throw a `ConfluentSchemaRegistryError` instead of a generic error, unless there's something higher up that catches it and re-throws.

Reply:
no I didn't check how the Java implementation does it.
I'll change it to `ConfluentSchemaRegistryError`